website/developer-docs/docs/writing-documentation.md

```diff
@@ -28,48 +31,27 @@ The site is built using npm, below are some useful make commands:
     For real time viewing of changes, as you make them.
 
 :::info
-Be sure to run the formatter before committing changes.
+Be sure to run a formatting command before committing changes.
```

Comment:
"... _the_ formatting command..."  We have specific formatting commands:

- `make lint-fix` (for Django)
- `make lint` (for Go and for other Python sources)
- `cd web ; npm run prettier` (for the WebUI)
- `cd website; npm run prettier` (for the web docs)

Reply:
It's possible I misunderstand the purpose of this document.   For our product documentation (`./website/`), we have the `npm run prettier` command which will format your documents for you.  (I'm surprised we don't run codespell or any other spell-check in there.). We have tools to _enforce_ styling, and we should use more of them.

Reply:
Yeah, about spell-check... I think we do run something, but whatever it is, is not as robust as what is in our final build pipeline! We will learn more soon, I hope. ;-)